src/features/account/AccountsTopNav.tsx

```diff
@@ -102,14 +104,16 @@ const AccountsTopNav = ({ onPressWallet, onLayout }: Props) => {
         paddingRight="l"
         onLayout={setBarButtonsRightWidth}
       >
-        <TouchableOpacityBox
-          paddingVertical="ms"
-          paddingLeft="s"
-          onPress={handleNotificationsSelected}
-          marginRight="s"
-        >
-          <NotificationIcon />
-        </TouchableOpacityBox>
+        {l1Network === 'helium' && (
```

Comment:
Are we only showing notifications for helium?

Reply:
We are only showing notifications button on the accounts screen for helium. If we are on Solana preview then we have a tab for that